spec/01-unit/02-conf_loader_spec.lua

```diff
@@ -341,6 +341,34 @@ describe("Configuration loader", function()
           assert.True(helpers.path.isabs(conf.ssl_cert))
           assert.True(helpers.path.isabs(conf.ssl_cert_key))
         end)
+        it("defines ssl_ciphers by default", function()
+          local conf, err = conf_loader(nil, {})
+          assert.is_nil(err)
+          assert.matches(":", conf.ssl_ciphers) -- looks kinda like a cipher suite
```

Comment:
`assert.matches(":", conf.ssl_ciphers, nil, true)` might be preferable here. Same for other `assert.matches` in this suite.